model/MediaSource.php

```diff
@@ -428,4 +434,13 @@ private function getPermissionsMapper(): MediaBrowserPermissionsMapper
 
         return $this->permissionsMapper;
     }
+
+    public function __destruct()
+    {
+        foreach ($this->tmpFiles as $tmpFile) {
+            if (file_exists($tmpFile) && is_writable($tmpFile)) {
```

Comment:
```suggestion
            if (is_writable($tmpFile)) {
```
As https://www.php.net/manual/en/function.is-writable.php `Returns true if the filename exists and is writable`